contract/models/contract_line.py

```diff
@@ -624,6 +641,12 @@ def _insert_markers(self, first_date_invoiced, last_date_invoiced):
         name = self.name
         name = name.replace("#START#", first_date_invoiced.strftime(date_format))
         name = name.replace("#END#", last_date_invoiced.strftime(date_format))
+        name = name.replace(
```

Comment:
Why not using Python for getting the localized month instead of adding more translations?

https://stackoverflow.com/a/17902535

Reply:
Hi Pedro,
it works fine!
I'm always afraid to change locale, even if for an instant, but if it's tested and doesn't affect other behaviors, it's fine, thanks